nixos/modules/services/networking/postgrest.nix

```
@@ -0,0 +1,103 @@
+{ lib, pkgs, config, ... }:
+let
+  cfg = config.services.postgrest;
+  inherit (lib) mkOption types literalExample;
+in
+{
+  options.services.postgrest = {
+    enable = lib.mkEnableOption "postgrest";
+    postgresUser = mkOption {
+      type = types.string;
+      example = "root";
+      description = ''
+        Username for postgres admin.
+      '';  
+    };
+    postgresPassword = mkOption {
+      type = types.string;
+      default = "";
+      example = "password";
+      description = ''
+        Password for postgres admin.
+      '';
+    };
+    postgresHost = mkOption {
+      type = types.string; 
+      default = "localhost"; # TODO: set host to config.services.postgres
+      example = "localhost";
+      description = ''
+        Host for the database.
+      '';
+    };
+    postgresSchema = mkOption {
+      type = types.string; 
+      default = "public"; # TODO: set host to config.services.postgres
+      example = "api";
+      description = ''
+        Schema of the database.
+      '';
+     };
+    postgresPort = mkOption {
+      type = types.int;
+      default = config.services.postgresql.port;
+      example = "5432";
+      description = ''
+        Port for the database.
+      '';
+    };
+    port = mkOption {
+      type = types.int;
+      default = 8080;
+      example = 8080;
```

```suggestion
      default = 5433;
```
examples should not repeat default. Also 8080 is common for web servers.